projects/core/src/store-finder/config/store-finder-config.ts

```diff
@@ -0,0 +1,8 @@
+export abstract class StoreFinderConfig {
```

Comment:
I think we should use `interface` here, instead of `abstract class`

Reply:
also we could optimize store finder config by moving `default-store-finder-config.ts` contents to this file. so we would have both interface and default value here. + imports would be nicer looking

Reply:
We can, but maybe let's keep the same convention of config files in core. As you can see for example `core/src/auth/config`, `core/src/occ/config`, `core/src/cms/config` etc. always consists of two files and the structure is defined as `abstract class`. In my opinion that make sense, because config structure is easily extendable.